test/e2e/deploy/kuma_deploy_global_zone.go

```diff
@@ -151,5 +176,24 @@ metadata:
 			Expect(err).ToNot(HaveOccurred())
 			return output
 		}, "5s", "500ms").Should(ContainSubstring("kuma-2-zone.demo-client"))
+
+		policy_create := trafficRoutePolicy(KumaNamespace, "traffic-default", 100)
+		policy_update := trafficRoutePolicy(KumaNamespace, "traffic-default", 101)
+
+		// Deny policy CREATE on zone
+		err := k8s.KubectlApplyFromStringE(c2.GetTesting(), c2.GetKubectlOptions(), policy_update)
+		Expect(err.Error()).To(ContainSubstring("should be only applied on global"))
+
+		// Accept policy CREATE on global
+		err = k8s.KubectlApplyFromStringE(c1.GetTesting(), c1.GetKubectlOptions(), policy_create)
+		Expect(err).ToNot(HaveOccurred())
+
+		// Deny policy UPDATE on zone
```

Comment:
I don't think you are checking update if you immediately do kubectl apply. It takes a couple of seconds to sync policy from global to remote. I'd first wait for the policy to be propagated.